docs/src/constants/redirect-routes.js

```diff
@@ -135,7 +135,6 @@ export default {
   '/components/web/tickets': ROUTES.TICKET,
   '/components/web/tooltips': ROUTES.TOOLTIP,
   '/components/web/typography': ROUTES.TEXT,
-  '/resources': ROUTES.GETTING_STARTED,
```

Comment:
Was simpler to remove this redirect than figure out how to dynamically populate it. The redirects are really niche anyway so I doubt this ever gets hit.

Reply:
These are essentially just strings though 🤔 
So you could do `'/resources': '/using',`

Reply:
Yeah I suppose we could just to `ROUTES.using` but I would wager that nobody has hit this route in 2020 so I don't think the redirect is still necessary

Reply:
Yeah that's fair

Reply:
We can call it 'ruthless prioritisation' 😉